### PR TITLE
Provide basic Apache Camel stack

### PR DESCRIPTION
**Pull request by apupier**
Signed-off-by: Aurélien Pupier <apupier@redhat.com>

### CODE REVIEW

**Comment by riuvshin** [comment]
Can one of the admins verify this patch?

**Comment by riuvshin** [comment]
Can one of the admins verify this patch?

**Comment by riuvshin** [comment]
Can one of the admins verify this patch?

**Comment by apupier** [comment]
ci-build

**Comment by apupier** [comment]
ci-test

**Comment by benoitf** [comment]
ci-build

File: ide/che-core-ide-stacks/src/main/resources/stacks.json [hunk 1]
```diff
@@ -1931,5 +1931,116 @@
       "name": "type-ceylon.svg",
       "mediaType": "image/svg+xml"
     }
-  }
+  },
```
Comment (benoitf) [inline ide/che-core-ide-stacks/src/main/resources/stacks.json]: FYI tabs need to be replaced by 2 spaces

**Comment by apupier** [comment]
ci-build

**Comment by apupier** [comment]
ci-test

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/816/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/816//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11245
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Comment by apupier** [comment]
ci-test

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/830/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/830//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11245
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Comment by apupier** [comment]
Can someone review please?

The PR is adding a new stack for Apache Camel which contains for now:
- activation of Language Server protocol
- things to do springboot development (based on the springboot stack)
- commands to help triggering springboot actions

the test provided is covering:
- ensure there is a stack and that a workspace can be created with it
- ensure that the Apache Camel Language Server is well started
it does not cover the commands

**Comment by benoitf** [comment]
@musienko-maxim ?

**Review by musienko-maxim — APPROVED** [review]

**Review by ashumilova — APPROVED** [review]

**Review by benoitf — APPROVED** [review]

**Pull request merged**